evalml/tests/objective_tests/test_fraud_detection.py

```diff
@@ -116,35 +116,29 @@ def test_binary_more_than_two_unique_values():
         fraud_cost.score(y_true, y_predicted)
 
 
-def test_fraud_objective_score(X_y_binary):
-    X, y = X_y_binary
+def test_fraud_objective_score():
     fraud_cost = FraudCost(amount_col="value")
 
-    y_predicted = pd.Series([0.1, 0.5, 0.5])
+    y_predicted = pd.Series([0.5, 0.1, 0.5])
```

Comment:
@chukarsten Previously, we thresholded in FraudCost by doing `predicted_proba * extra_columns`, and the thresholding was `5`. This meant the values we thresholded on looked like
```python
[0.1, 0.5, 0.5] * [100, 5, 200] = [10, 2.5, 100]
```
Thresholding with 5 on this would result in an array that matches `y_true`. Now that I changed thresholding to be on the `predicted_proba` directly, I needed to change the `y_predicted` values to then match `y_true` post-thresholding with the 0.45 threshold.

Reply:
Awesome, thanks for the explanation!